src/sentry/static/sentry/app/components/charts/pieChart.jsx

```diff
@@ -68,6 +68,20 @@ class PieChart extends React.Component {
     });
   };
 
+  // echarts Legend does not have access to percentages (but tooltip does :/)
+  getSeriesPercentages = series => {
+    const total = series.data.reduce((acc, {value}) => acc + value, 0);
+    return series.data
+      .map(({name, value}) => [name, Math.round(value / total * 10000) / 100])
+      .reduce(
```

Comment:
seems like you could just skip the `map` and do 

```
.reduce(acc, {name, value}) => ({
  ...acc,
  [name]: Math.round(value / total * 10000) / 100
}, {})
```

tho maybe there are performance benefits to `map` you are going after?